specification/cognitiveservices/OpenAI.Inference/models/images.tsp

```diff
@@ -0,0 +1,137 @@
+
+  @doc("A unique identifier representing your end-user, which can help to monitor and detect abuse.")
+  user?: string;
+}
+
+@doc("The image url if successful, and an error otherwise.")
+model ImageLocation {
+  @doc("The URL that provides temporary access to download the generated image.")
+  url?: url;
+
+  @doc("The error if the operation failed.")
+  error?: Foundations.Error;
+}
+
+@doc("The result of the operation if the operation succeeded.")
+model ImageResponse {
+  @doc("A timestamp when this job or item was created (in unix epochs).")
+  created: int64;
+
+  @doc("The images generated by the operator.")
+  data: ImageLocation[];
+}
+
+@doc("The result of the operation if the operation succeeded.")
+model ImageOperationResponse {
+  @doc("The ID of the operation.")
+  id: string;
+
+  @doc("A timestamp when this job or item was created (in unix epochs).")
+  created: int64;
+
+  @doc("A timestamp when this operation and its associated images expire and will be deleted (in unix epochs).")
+  expires?: int64;
+
+  @doc("The result of the operation if the operation succeeded.")
+  result?: ImageResponse;
+
+  @doc("The status of the operation")
+  status: State;
+
+  @doc("The error if the operation failed.")
+  error?: Foundations.Error;
+}
+
+@lroStatus
+@doc("The state of a job or item.")
+enum State {
+  @doc("The operation was created and is queued to be processed in the future.")
+  notRunning,
+
+  @doc("The operation has started to be processed.")
+  running,
+
+  @lroSucceeded
+  @doc("The operation has successfully be processed and is ready for consumption.")
+  succeeded,
+
+  @lroCanceled
+  @doc("The operation has been canceled and is incomplete.")
+  canceled,
+
+  @lroFailed
+  @doc("The operation has completed processing with a failure and cannot be further consumed.")
+  failed,
+
+  @doc("The entity has been deleted but may still be referenced by other entities predating the deletion.")
+  deleted,
+}
+
+@doc("Provides status details for long running operations.")
+model ImageOperationStatus {
+  @key("operationId")
+  @visibility("read", "query")
+  @doc("The unique ID of the operation.")
+  id: string;
+
+  @doc("The status of the operation")
+  status: State;
+}
+
+#suppress "@azure-tools/typespec-azure-core/long-running-polling-operation-required" "template"
+@Foundations.Private.needsRoute
+@post
+@doc("Long running operation template to match Azure OpenAI .")
+op OaiLongRunningRpcOperation<
+  TParams extends TypeSpec.Reflection.Model,
+  TResponse extends TypeSpec.Reflection.Model,
+  TStatusResult extends TypeSpec.Reflection.Model,
+  TStatusError = Foundations.Error,
+  Traits extends TypeSpec.Reflection.Model = {},
+  TErrorResponse = Azure.Core.Foundations.ErrorResponse
+> is Foundations.Operation<
+  TParams &
+    Azure.Core.Traits.Private.TraitProperties<
+      Traits,
+      Azure.Core.Traits.TraitLocation.Parameters
+    >,
+  Azure.Core.Foundations.AcceptedResponse &
+    ImageOperationResponse &
```

Comment:
I think this is a typo, we should use `TResponse` instead of `ImageOperationResponse`.
```suggestion
    TResponse &
```